src/TConsole.cpp

```diff
@@ -2681,6 +2682,42 @@ bool TConsole::setBackgroundColor(const QString & name, int r, int g, int b, int
 
 }
 
+bool TConsole::raiseWindow(const QString & name )
+{
+    std::string key = name.toLatin1().data();
+    if( mSubConsoleMap.find( key ) != mSubConsoleMap.end() )
+    {
+        mSubConsoleMap[key]->raise();
```

Comment:
Not sure it needs to be the non-const `operator[]` here, the `.value()` one will give you the `const` pointer which I think will work faster/safer in this situation.